.devcontainer/docker-compose.yml

```diff
@@ -17,10 +17,10 @@ services:
       - /var/run/docker.sock:/var/run/docker.sock
 
     # Uncomment the next four lines if you will use a ptrace-based debugger like C++, Go, and Rust.
-    # cap_add:
-    #   - SYS_PTRACE
-    # security_opt:
-    #   - seccomp:unconfined
+    cap_add:
```

Comment:
This change (supporting ptrace-based debuggers) is not supported by any other changes in the PR (i.e. adding the needed libraries to support it as well)